<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>우편물 접수</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>title_big.png</normaloff>title_big.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : rgba(0,0,0,0%)</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'맑은 고딕'; font-size:14pt; font-weight:696;&quot;&gt;우편물 접수&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>접수날짜</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="datesel_start">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="datesel_end">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_loadpostlist">
          <property name="text">
           <string>조회</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="info_14_isprocessed">
          <property name="text">
           <string>결재된 우편물만 조회 </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1066</width>
           <height>150</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="7">
             <widget class="QLineEdit" name="info_9_horizontal">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QLineEdit" name="info_4_receiver"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="info_1_importer"/>
            </item>
            <item row="4" column="4">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>메모</string>
              </property>
             </widget>
            </item>
            <item row="2" column="9">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>우편중량</string>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QSpinBox" name="info_7_spin_quantity"/>
            </item>
            <item row="0" column="7">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>주소</string>
              </property>
             </widget>
            </item>
            <item row="4" column="7">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>세로길이</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="info_2_depart">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="2" column="7">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>가로길이</string>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QComboBox" name="info_8_combo_sort"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>접수자명</string>
              </property>
             </widget>
            </item>
            <item row="5" column="4" colspan="2">
             <widget class="QLineEdit" name="info_13_memo"/>
            </item>
            <item row="1" column="7" colspan="4">
             <widget class="QLineEdit" name="info_5_address"/>
            </item>
            <item row="3" column="9">
             <widget class="QLineEdit" name="info_11_weight"/>
            </item>
            <item row="5" column="8">
             <widget class="QComboBox" name="info_10_vertical_format"/>
            </item>
            <item row="0" column="5">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>받는사람</string>
              </property>
             </widget>
            </item>
            <item row="5" column="7">
             <widget class="QLineEdit" name="info_10_vertical">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="10">
             <widget class="QComboBox" name="info_11_weight_format"/>
            </item>
            <item row="2" column="4">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>수량</string>
              </property>
             </widget>
            </item>
            <item row="4" column="9">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>높이</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>보내는사람</string>
              </property>
             </widget>
            </item>
            <item row="5" column="10">
             <widget class="QComboBox" name="info_12_height_format"/>
            </item>
            <item row="5" column="9">
             <widget class="QLineEdit" name="info_12_height">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QLineEdit" name="info_3_sender">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>부서명</string>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>종류</string>
              </property>
             </widget>
            </item>
            <item row="3" column="8">
             <widget class="QComboBox" name="info_9_horizontal_format"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>사원번호</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="info_0_importerid"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>제목</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="4">
             <widget class="QLineEdit" name="info_6_title">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>전화번호</string>
              </property>
             </widget>
            </item>
            <item row="0" column="8">
             <widget class="QCheckBox" name="info_z_isurgent">
              <property name="text">
               <string>긴급여부</string>
              </property>
             </widget>
            </item>
            <item row="0" column="9">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>우편변호</string>
              </property>
             </widget>
            </item>
            <item row="0" column="10">
             <widget class="QLineEdit" name="info_14_postno"/>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QLineEdit" name="info_15_phonenum"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,0,0,0,0,0">
        <item row="0" column="4">
         <widget class="QPushButton" name="btn_delpostcolumn">
          <property name="text">
           <string>제거</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="btn_savepostlist">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>저장</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>338</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btn_modifypostcolumn">
          <property name="text">
           <string>수정</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btn_addpostcolumn">
          <property name="text">
           <string>추가</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_selectallrow">
          <property name="text">
           <string>전체선택</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>오늘자 접수된 우편물 목록</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="table_postlist">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>접수날짜</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>사원번호</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>접수자명</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>부서명</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>보내는사람</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>받는사람</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>주소</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>제목</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>수량</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>종류</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>긴급여부</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>가로길이</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>세로길이</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>우편중량</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>높이</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>메모</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>메뉴</string>
    </property>
    <addaction name="action_closewindow"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_closewindow">
   <property name="text">
    <string>창 닫기</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>datesel_start</tabstop>
  <tabstop>datesel_end</tabstop>
  <tabstop>btn_loadpostlist</tabstop>
  <tabstop>info_14_isprocessed</tabstop>
  <tabstop>info_0_importerid</tabstop>
  <tabstop>info_1_importer</tabstop>
  <tabstop>info_2_depart</tabstop>
  <tabstop>info_3_sender</tabstop>
  <tabstop>info_4_receiver</tabstop>
  <tabstop>info_5_address</tabstop>
  <tabstop>info_z_isurgent</tabstop>
  <tabstop>info_14_postno</tabstop>
  <tabstop>info_6_title</tabstop>
  <tabstop>info_7_spin_quantity</tabstop>
  <tabstop>info_8_combo_sort</tabstop>
  <tabstop>info_9_horizontal</tabstop>
  <tabstop>info_9_horizontal_format</tabstop>
  <tabstop>info_10_vertical</tabstop>
  <tabstop>info_10_vertical_format</tabstop>
  <tabstop>info_11_weight</tabstop>
  <tabstop>info_11_weight_format</tabstop>
  <tabstop>info_12_height</tabstop>
  <tabstop>info_12_height_format</tabstop>
  <tabstop>info_15_phonenum</tabstop>
  <tabstop>info_13_memo</tabstop>
  <tabstop>btn_modifypostcolumn</tabstop>
  <tabstop>btn_addpostcolumn</tabstop>
  <tabstop>btn_delpostcolumn</tabstop>
  <tabstop>btn_savepostlist</tabstop>
  <tabstop>btn_selectallrow</tabstop>
  <tabstop>table_postlist</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
